File: longestCommonPrefix.py
def longestCommonPrefix(strs):
    pref=""

    for i in range(len(strs[0])):
        for word in strs:
            if i == len(word) or word[i] != strs[0][i]:
                return pref
        pref += strs[0][i]
    return pref

File: C#/Easy/longestCommonPrefix.cs
public class Solution {
    public string LongestCommonPrefix(string[] strs) {
        if (strs.Length == 0) return "";
       Array.Sort(strs, (x, y) => x.Length.CompareTo(y.Length));
        string prefix = "";
        for(int i = 0; i < strs[0].Count(); i++)
        {
            for(int j = 1; j < strs.Count(); j++)
            {
                if (strs[j][i] != strs[0][i])
                    return prefix;
            }
            prefix += strs[0][i];
        }
        return prefix;
    }
}